module/VuFind/src/VuFind/Controller/HierarchyController.php

```diff
@@ -177,7 +177,8 @@ public function gettreejsonAction()
         }
 
         // If we got this far, something went wrong:
-        return $this->outputJSON('error', 503); // Service Unavailable
+        return $this->outputJSON('{"error": {"code": 503, "message": "Service Unavailable"}}', 503); 
```

Comment:
I see that this is causing a CI failure due to line length. You could either wrap the line like:

```
return $this->outputJSON(
    '{"error": {"code": 503, "message": "Service Unavailable"}}',
    503
);
```

Or you might consider structuring the response data in PHP and then encoding it:

```

$response = ['error' => ['code': 503, 'message' => 'Service Unavailable']];
return $this->outputJSON(json_encode($response), 503);
```

Reply:
i did the second variant, it seems a bit more clear and saves us the hassle of manually typing json in a php document